### DISCUSSION TITLE
Need a team for a megaproject

### DISCUSSION THREAD

**Discussion by Team-Oneninet** [TurboWarp/extensions#1317] [General]
I'm pushing the Scratch engine and my sanity to the test by making a working OS/VM in Turbowarp.

If you are interested, info here: https://scratch.mit.edu/projects/970482863/

Thank you.

**Comment by milkyestway** [comment]
Im down and recommend you look at things like linux on scratch, I also made an interpreter to make console commands easier if you want it.
